res/config.json

```diff
@@ -3,18 +3,21 @@
     {
       "42": {
         "trunks": [
-          {
-            "42": {
-              "stations": [
-                {
-                  "endpoint": "SIP/100"
-                },
-                {
-                  "endpoint": "SIP/96@cwolfe"
-                }
-              ]
-            }
-          }
+          "42-A"
+        ],
+       "stations": [
+         "SIP/bobmarley",
+         "SIP/kingcrimson"
+        ]
+      }
+    },
+    {
+      "420": {
+        "trunks": [
+          "42-A"
+        ],
+       "stations": [
```

Comment:
2 is more common in JavaScript code bases. They are more than welcomed to use 4 of course, but figured I would mention it since my coding style probably has influenced them but in this case the indenting is incorrect regardless.